components/playlist/browser/resources/components/playlistPlayer.tsx

```diff
@@ -6,17 +6,99 @@
 import * as React from 'react'
 import { useSelector } from 'react-redux'
 import { RouteComponentProps, Redirect } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { color, radius } from '@brave/leo/tokens/css'
+import LeoButton from '@brave/leo/react/button'
 
 import VideoFrame from './videoFrame'
 import PlaylistItem from './playlistItem'
-import { ApplicationState, PlayerState, usePlaylist } from '../reducers/states'
+import {
+  ApplicationState,
+  PlayerState,
+  PlaylistEditMode,
+  usePlaylist,
+  usePlaylistEditMode
+} from '../reducers/states'
 import postMessageToPlayer from '../api/playerApi'
 import { types } from '../constants/playlist_types'
+import { getAllActions } from '../api/getAllActions'
+import { getPlaylistAPI } from '../api/api'
 
 interface MatchParams {
   playlistId: string
 }
 
+const StyledEditActionContainer = styled.div`
+  display: flex;
+  padding: 16px;
+  align-items: center;
+  gap: 16px;
+  align-self: stretch;
+  height: 68px;
+  border-bottom: 1px solid ${color.divider.subtle};
+`
+
+const StyledEditButton = styled(LeoButton)`
+  width: fit-content;
+  flex-grow: 0;
+  --leo-button-padding: 10px 12px;
+  --radius: ${radius.full};
```

Comment:
You might be able to set the `fab` property on the Leo button instead, if you're just trying to get a round button:

https://leo.bravesoftware.com/?path=/story/button--fab&args=kind:plain-faint;size:small

Reply:
Oh, that's cool. But actually this doesn't have any effect. I think we need to modify css in the leo button like `--radius= var(--leo-button-radius, 16px)`. Removed.